Library/Homebrew/bintray.rb

```diff
@@ -106,10 +112,18 @@ def upload_bottle_json(json_files, publish_package: false)
     formula_packaged = {}
 
     bottles_hash.each do |formula_name, bottle_hash|
+      formula = Formula[formula_name]
       version = ERB::Util.url_encode(bottle_hash["formula"]["pkg_version"])
```

Comment:
I don't think these versions will match, unfortunately, so this won't work if a new `mirror` is added to the formula. Maybe this isn't a big deal, though. @issyl0 and others: thoughts?

Reply:
For the cases about which I wrote the original issue, I didn't even consider people adding new mirrors. Just that I saw existing ones in formulae that needed a manual step. So I think this is fine. If new things aren't mirroring automatically as we expect, we can rework this later.